### DISCUSSION TITLE
rtabmap python d435i

### DISCUSSION THREAD

**Discussion by yash7321** [introlab/rtabmap#1310] [Q&A]
can i use rtabmap if i want to do slam with only d435i {depth(pointcloud) and imu}camera without using ros, if so what steps do i need to follow

**Comment by matlabbe** [comment]
We don't have a python wrapper of the library, but you can do it in c++: https://github.com/introlab/rtabmap/wiki/Cplusplus-RGBD-Mapping

or just use the standalone application: https://github.com/introlab/rtabmap/releases
